org.eclipse.xtext.xbase.junit/xtend-gen/org/eclipse/xtext/xbase/compiler/InMemoryJavaCompiler.java

```diff
@@ -238,7 +238,7 @@ public InMemoryJavaCompiler(final ClassLoader parent, final CompilerOptions comp
     InMemoryJavaCompiler.ClassLoaderBasedNameEnvironment _classLoaderBasedNameEnvironment = new InMemoryJavaCompiler.ClassLoaderBasedNameEnvironment(parent);
     this.nameEnv = _classLoaderBasedNameEnvironment;
     this.parentClassLoader = parent;
-    Map _map = compilerOptions.getMap();
+    Map<String, String> _map = compilerOptions.getMap();
```

Comment:
I think that also this change is due to some internal changed in JDT which are only in the latest versions, so this code would not be runnable in an older Eclipse version (see also the other comment)